src/pages/office_supplies/OfficeSupplies.java

```diff
@@ -0,0 +1,38 @@
+package pages.office_supplies;
+
+import jdk.nashorn.internal.runtime.regexp.JoniRegExp;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+/**
+ * Created by ddantas on 4/4/2017.
+ */
+public class OfficeSupplies {
+
+    protected WebDriver driver;
+
+    public OfficeSupplies(WebDriver driver)
+    {
+        this.driver = driver;
+
+        PageFactory.initElements(driver,this);
```

Comment:
No. It's a new sub heading under "shop by category" which has 20 more subheading.